Gemfile

```diff
@@ -19,6 +19,7 @@ gem "faker", require: false
 gem "faraday", require: false
 gem "govuk_design_system_formbuilder"
 gem "grover"
+gem "httparty"
```

Comment:
we already have [faraday](https://github.com/unboxed/bops/blob/main/Gemfile#L19) in the project. How about trying to use Faraday to make the `POST` request?